### ISSUE TITLE
Font error when loading astrosymb in Overleaf

### ISSUE THREAD

**Issue by jmangum** [AASJournals/AASTeX60#106]
Starting with the aastex63.tex template to create a new article, I find that when I include astrosymb in my documentclass command:

\documentclass[twocolumn,astrosymb,tighten,trackchanges]{aastex63}

I get a symbol error.  See attached screenshot of the error message.

-- Jeff
![Screen Shot 2020-01-27 at 2 50 08 PM](https://user-images.githubusercontent.com/3649096/73208478-84b7a280-4114-11ea-98b7-bdd2db583309.png)

**Comment by augustfly** [comment]
Hi Jeff,

What did you upgrade from? I can't see any code change from v6.1 to v6.3 related to astrosym(b).  There was a slight tweak from v6.0 to 6.1 but we've never gotten a bug report about it before. 

Honestly, this looks to me like Overleaf can't find the necessary font file, which would be something for them to fix.

Can you ask them about this and if they can't fix it then we can try again to determine what's changed?

**Comment by jmangum** [comment]
Thanks Gus.  I agree that it looks like an Overleaf problem (I have not tried it on my local latex installation).  Will try to figure out how to bring it to Overleaf's attention.

**Comment by jmangum** [comment]
Overleaf support responded to my trouble ticket.  See below.  Seems that they think this is an AASTex problem...

=============
Hi,

Thanks for getting in touch, and sorry that you are having this issue. 

From what I can find, the astrosymb fonts are not freely available, so are not included in the standard TeXLive distribution. They also do not seem to be included with the full package downloaded from AAS (https://journals.aas.org/aastexguide/).

Looking around, I found this twitter conversation with AAS about this option: https://twitter.com/astro_wright/status/772637107122794501 - not much to go on, I am afraid.

In principle, if you purchase the astrosymb fonts and include them in your project, this option may allow you to use them. It sounds that this is just an optional enhancement, should you happen to own and want to use the astrosymb fonts. If AAS is making these fonts available to authors, you may need to follow up with them to find out about where to obtain them.

Hope this helps.

Best,

Dan

—
Dan MacKinnon
Overleaf Product Specialist

**Comment by augustfly** [comment]
I believe the overleaf response is entirely incorrect. But the actual problem may depend heavily on your particular TeX install.  Here is my install information (please provide yours before we continue talking this through!):

```
> latex --version
pdfTeX 3.14159265-2.6-1.40.18 (TeX Live 2017)
kpathsea version 6.2.3
Copyright 2017 Han The Thanh (pdfTeX) et al.
There is NO warranty.  Redistribution of this software is
covered by the terms of both the pdfTeX copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the pdfTeX source.
Primary author of pdfTeX: Han The Thanh (pdfTeX) et al.
Compiled with libpng 1.6.29; using libpng 1.6.29
Compiled with zlib 1.2.11; using zlib 1.2.11
Compiled with xpdf version 3.04

```
First, I confirmed your error locally on my desktop. Second, I confirmed that **a**strosym the font is installed on my desktop. It is. But **A**strosym is not.  If I changed lines 201-203:

```
 \font\astro@font=astrosym at 8pt
\else
 \font\astro@font=astrosym at 7pt

```
to lower case then the compiler found the font install and used it correctly. Note on my machine the astrosym font is installed here:

`/usr/local/texlive/2017/texmf-dist/fonts/tfm/public/astro/astrosym.tfm`

**Comment by jmangum** [comment]
Thanks Gus.  Note that I am doing all of this completely on overleaf.  Have not tried anything with a local TeX installation.  Even though it is not relevant to this issue, I have the same TeX Live 2017 install as you.

**Comment by augustfly** [comment]
Oh right!  You could upload a modified aastex63.cls to overleaf (that fixes the capitalization problem) to see if they have the font but their compiler doesn't see it because of the capital "A".  I'm pretty sure the uploaded aastex63.cls (see attached version) overrides the built in Overleaf one. 

[aastex63-astrosym.zip](https://github.com/AASJournals/AASTeX60/files/4129110/aastex63-astrosym.zip)

**Comment by jmangum** [comment]
ok.  Before I try the new cls file, though, note that I started with the AASTeX 6.3 template in overleaf, which includes an aastex63.cls file in the overleaf project directory (along with the latex file, the Orcid-ID.png file, and a figure file directory).  By "built in" do you mean the aastex63.cls file provided with the overleaf AASTeX template?

**Comment by augustfly** [comment]
Yes that one. My proposed test is this: replace or edit that aastex63.cls file so that it uses lower case for astrosym (lines 201-203).  Then try the compile. 

If that still doesn't work then download the .log file, zip it, and upload it here, and I'll take a look at the problem specific to Overleaf.

**Comment by jmangum** [comment]
Indeed changing "Astrosym" to "astrosym" on lines 201-203 did indeed fix the problem.  Thanks!
